evap/evaluation/auth.py

```diff
@@ -133,16 +135,28 @@ def reward_user_required(user):
 
 # see https://mozilla-django-oidc.readthedocs.io/en/stable/
 class OpenIDAuthenticationBackend(OIDCAuthenticationBackend):
+    @staticmethod
+    def possible_existing_account_emails(email: str) -> Iterable[str]:
+        yield email
+        name, domain = email.split("@", maxsplit=1)
```

Comment:
TIL: email addresses can contain multiple `@` characters, if they are "quoted"

> Space and special characters "(),:;<>@[\] are allowed with restrictions (they are only allowed inside a quoted string...)

In fact, django uses `rsplit("@", maxsplit=1)` to get the domain part: https://github.com/django/django/blob/0f5dd0dff3049189a3fe71a62670b746543335d5/django/core/validators.py#L250

```suggestion
        name, domain = email.rsplit("@", maxsplit=1)
```

Reply:
Yeah we also thought about that on Monday, but I thought that a) we do this wrong in other places in the codebase and b) this is only wrong if the settings contain a key with an @ in it, which would be wrong according to spec. I wanted to follow up on this separately